Sources/Rendering/Core/Picker/index.js

```diff
@@ -209,8 +214,8 @@ function vtkPicker(publicAPI, model) {
         scale[1] = vtkMath.norm(col2);
         scale[2] = vtkMath.norm(col3);
 
-        vec3.transformMat4(p1Mapper, p1World, model.transformMatrix);
-        vec3.transformMat4(p2Mapper, p2World, model.transformMatrix);
+        vec4.transformMat4(p1Mapper, p1World, model.transformMatrix);
+        vec4.transformMat4(p2Mapper, p2World, model.transformMatrix);
 
```

Comment:
Do you need divide p*Mapper by p*Mapper[3] here? I think you should to be safe.

Reply:
Ping

Reply:
Yes, sorry I just made the correction after your response. But yes, to be more efficient and understandable, it should be safe do divide pMapper by pMapper[3].